fuse_core/include/fuse_core/ceres_options.h

```diff
@@ -0,0 +1,188 @@
+/***************************************************************************
+ * Copyright (C) 2019 Clearpath Robotics. All rights reserved.
```

Comment:
This license is probably going to have to change.

Reply:
No problem. Would you guys change it before merging?